_posts/2022-11-25-async-sink-rate-limiting-strategy.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Optimising the throughput of async sinks using a custom RateLimitingStrategy"
+date: 2022-11-25T12:00:00.000Z
+authors:
+- liangtl:
+  name: "Hong Liang Teoh"
+excerpt: An overview of how to optimise the throughput of async sinks using a custom RateLimitingStrategy
+---
+
+## Introduction
+
+When designing a Flink data processing job, one of the key concerns is maximising job throughput. Sink throughput is a crucial factor because it can determine the entire job’s throughput. We generally want the highest possible write rate in the sink without overloading the destination. However, since the factors impacting a destination’s performance are variable over the job’s lifetime, the sink needs to adjust its write rate dynamically. Depending on the sink’s destination, it helps to tune the write rate using a different RateLimitingStrategy.
```

Comment:
Given that the docs style guide https://flink.apache.org/contributing/docs-style.html says we should use US English, you should change `maximising` to `maximizing`.